recipes/b2/portable/test_package/conanfile.py

```diff
@@ -1,8 +1,7 @@
-from conans import ConanFile, tools
-import os
+from conan import ConanFile
 
 
-class TestPackgeConan(ConanFile):
+class TestPackageConan(ConanFile):
     settings = "os", "arch"
 
     def test(self):
```

Comment:
For v2 there's a few items that are missing, you should be able to copy the template  https://github.com/conan-io/conan-center-index/blob/e8ba8ec47cae8e50d09fe4afe5e889c8698b7c42/docs/package_templates/prebuilt_tool_package/all/test_package/conanfile.py#L11
And replace the run command 🤞